### ISSUE TITLE
How was the pre-trained model obtained?

### ISSUE THREAD

**Issue by zxd-cqu** [Zj-BinXia/SSL#6]
I debugged the code provided by the official source and found that the pre-trained model is also built using modules like Conv2D_WN_Pre and Conv2D_WN. Can I understand it as follows:  Both the pre-trained model and the initially constructed network in the pruning process are derived from the network defined in SSL-master/basicsr/archs/basicvsr_arch.py. If this understanding is correct, how was the pre-trained model trained? What was the loss function, and was the training applied to both scaling factors and convolutional weights in the same way?
Is it trained directly, like in original basicvsr, without any additional operations?

**Comment by Zj-BinXia** [comment]
You don't have to think too complicatedly. The training method is based on Basicsr, not on pruning.

**Comment by zxd-cqu** [comment]
So the pre-trained model do including Conv2D_WN_Pre and Conv2D_WN modules along with trainable scaling factors. I can train it just directly using the methods provided in basicvsr.

**Comment by Zj-BinXia** [comment]
Yes

**Comment by zxd-cqu** [comment]
Got it. Thank you for your response.

**Comment by sunyclj** [comment]
> So the pre-trained model do including Conv2D_WN_Pre and Conv2D_WN modules along with trainable scaling factors. I can train it just directly using the methods provided in basicvsr.

if  the pre-trained model  is not included Conv2D_WN_Pre and Conv2D_WN with trainable scaling factors, can't it?
